habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -8,48 +8,47 @@
 See README.md in this directory.
 """
 
-import ctypes
-
-# must call this before importing habitat or magnum! avoids EGL_BAD_ACCESS error on some platforms
```

Comment:
Right now, this still causes the issue.

This can be fixed by moving `from habitat_hitl.core.hitl_main import hitl_main` above `import magnum as mn` in the applications (e.g. `basic_viewer.py`), or adding that line everywhere.

I wonder if it would be simpler to include it to `magnum` and `habitat_sim` `__init__.py`.